### PR TITLE
chore: Format with standard

### PR DESCRIPTION
**Pull request by alxckn**
## This PR

OpenFeature SDK lints using standardrb, PR to homogenize in the flagd provider

### CODE REVIEW

File: providers/openfeature-flagd-provider/.rubocop.yml
Comment (beeme1mr) [inline providers/openfeature-flagd-provider/.rubocop.yml:1]: Would it be possible to move the lining rules to the root of the repo so they could be shared with other packages in the future?

**Review by beeme1mr — APPROVED** [review]
Looks good, thanks!

File: providers/openfeature-flagd-provider/openfeature-flagd-provider.gemspec [hunk 1]
```diff
@@ -23,13 +23,13 @@ Gem::Specification.new do |spec|
...
+  spec.bindir = "exe"
+  spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
   spec.add_runtime_dependency "grpc", "~> 1.50"
 
   spec.add_development_dependency "rake", "~> 13.0"
   spec.add_development_dependency "rspec", "~> 3.12.0"
-  spec.add_development_dependency "rubocop", "~> 1.37.1"
+  spec.add_development_dependency "standard", "~> 1.35.1"
```
Comment (maxveldink) [inline providers/openfeature-flagd-provider/openfeature-flagd-provider.gemspec:34]: nitpick: I think it's safe to pessimistic lock the minor version here to get all new rule changes.

```suggestion
  spec.add_development_dependency "standard", "~> 1.35"
```

**Review by maxveldink — APPROVED** [review]
🎉

File: providers/openfeature-flagd-provider/.rubocop.yml
[the remark this answers is not in the document]
Comment (alxckn) [reply providers/openfeature-flagd-provider/.rubocop.yml:1]: It's a personal addition, standardrb doesn't seem to work with my IDE setup (ruby-lsp)

Comment (alxckn) [reply providers/openfeature-flagd-provider/.rubocop.yml:1]: I've made a change to use rubocop's `inherited_from` configuration to move the configuration to a shared config folder in my last commit (cf. https://docs.rubocop.org/rubocop/configuration.html)

**Pull request merged**